FeedStoreChallenge/RealmFeedStore.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  RealmFeedStore.swift
+//  FeedStoreChallenge
+//
+//  Created by Carlos Damiao on 21/01/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+
+public class RealmFeedStore: FeedStore {
+
+	private let realm: Realm
+
+	public init(config: Realm.Configuration) {
+		realm = try! Realm(configuration: config)
+	}
```

Comment:
`try!` can lead to runtime errors - ideally, this initializer should throw errors so clients can handle them.

```suggestion
	public init(config: Realm.Configuration) throws {
		realm = try Realm(configuration: config)
	}
```